trino/sqlalchemy/dialect.py

```
@@ -229,16 +229,20 @@ def get_check_constraints(
         return []
 
     def get_table_comment(self, connection: Connection, table_name: str, schema: str = None, **kw) -> Dict[str, Any]:
-        properties_table = self._get_full_table(f"{table_name}$properties", schema)
-        query = f'SELECT "comment" FROM {properties_table}'
+        query = dedent(
```

It would be better to raise a error when `schema` is None in my understanding.